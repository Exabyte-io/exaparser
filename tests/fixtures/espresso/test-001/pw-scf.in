version https://git-lfs.github.com/spec/v1
oid sha256:848dfaabaff0efa6bc44d3180839ae284a3d41c9c054d2666017c98746ae11bb
size 884
