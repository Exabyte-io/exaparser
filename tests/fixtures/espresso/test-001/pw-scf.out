version https://git-lfs.github.com/spec/v1
oid sha256:c31175bda22862bfd62acc454d630d5f8102de2b0f097e5fe48296d8c8cb0cf9
size 11496
